bitnami/postgresql-ha/templates/postgresql/statefulset.yaml

```
@@ -72,6 +72,9 @@ spec:
       {{- if .Values.topologySpreadConstraints }}
       topologySpreadConstraints: {{- include "common.tplvalues.render" (dict "value" .Values.topologySpreadConstraints "context" .) | nindent 8 }}
       {{- end }}
+      {{- if .Values.postgresql.hostNetwork }}
+      hostNetwork: {{ .Values.postgresql.hostNetwork }}
+      {{- end }}
```

You can simplify this since the parameter is a boolean
```suggestion
      hostNetwork: {{ .Values.postgresql.hostNetwork }}
```